pitch.md

```diff
@@ -26,38 +111,100 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+There are several other DSLs in the domain. Milo Han has a small set of links [http://www.randomsheetmusic.com/][Random Sheet Music] Randomly generates music for single clef instruments.
+
+[http://www.link.cs.cmu.edu/melody-generator/][Melisma Stochastic Melody Generator ] Generates midi files based on constraints given by the users.
+
+[https://www.cs.hmc.edu/~keller/jazz/improvisor/][Impro-visor] Generates single line jazz improvisations over lead sheets. It can be used to practice jazz improvisation.
+
+[https://www.sightreadingfactory.com/][Sight Reading Factory] Commerical software that generates surprisingly good sheet music. Users pay to use it.
+
+The algorithms behind generating the music in these places are sophisticated. However, the ability
+to describe what a user wants is not in my opinion. A music teacher teaching students of all levels
+would need the ability to describe a lot of kinds of things about music in order to give good
+sight reading material.
 
 ## Suitability
 _What percentage of your time do you think will be spent directly engaging in
 the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+I would like to focus more time on language design because I feel that is an important
+part of this language. My hope would be something like 60-40 or 70-30 in favor of
+language design.
+
+I know that creating the algorithms will take a lot of time. For now, I think I will just
+do my best and won't strive to create perfect algorithms.
 
 ## Scope
 _How big an idea is this? How ambitious is this project?_
 
+I think this is reasonable. It focuses mainly on how musicians of different levels express
+the kind of music they want to sight read. This project will probably be a proof-of-concept
+because the algorithms are difficult to create well. 
 
 ## Challenges and opportunities
 _Why is this a good project? What are some challenges you expect to face? How
 might you overcome them? What are some warning signs that the project has gotten
 off track, and how will you get the project back on track if needed?_
+
+This is an interesting project because I feel like there can be improvements in how musicians
+communicate their desires to computers and DSLs. I think with help from the people in the class
+and people from the domain, we can create a DSL that allows musicians to express the right kind
+of music for sight reading. The biggest warning sign is the case where I spend most of my time
+implementing algorithms to generate the random music. This project should be mainly focused
+about language design, but of course there will be some time focused on implementing the algorithms
+themselves. 
```

Comment:
Exactly. I think you can totally put a lot of focus on creating a really interesting language design that will really help musicians everywhere get better at sight reading and I think you can find the balance between implementing random music generators and the language design.